attest/attest.go

```diff
@@ -111,6 +155,17 @@ type PlatformEK struct {
 	Public crypto.PublicKey
 }
 
+// AIKParameters describes information about an AIK. This information
+// is typically used to generate an activation challenge.
+type AIKParameters struct {
+	Version TPMVersion
```

Comment:
I don't like that there are multiple ways to get the TPMVersion, can you drop this and continue to have users call `func (t *TPM) Info()`?

Reply:
Info() results in a roundtrip to the TPM to retrieve a whole bunch of fields, so I would rather not do that for trivial fields like version.
Maybe we can have a `tpm.Version()` instead, given that TPMVersion is a field on the TPM struct?